packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -138,8 +140,26 @@ export const _RemoteVideoTile = React.memo(
       );
     }, [renderElement, showLoadingIndicator]);
 
+    /* @conditional-compile-remove(pinned-participants) */
+    const onKeyDown = useCallback(
+      (e: KeyboardEvent) => {
+        if (e.key === 'Enter') {
+          setDrawerMenuItemProps(
+            convertContextualMenuItemsToDrawerMenuItemProps(contextualMenuProps, () => setDrawerMenuItemProps([]))
+          );
+        }
+      },
+      [setDrawerMenuItemProps, contextualMenuProps]
+    );
+
     return (
-      <Stack style={remoteVideoTileWrapperStyle}>
+      <Stack
+        /* @conditional-compile-remove(pinned-participants) */
+        tabIndex={remoteVideoTileMenuKind === 'drawer' ? 0 : undefined}
```

Comment:
why are we setting the tab order of the video tile to `0` when it will show a drawer? it shouldn't be any different for mobile incase the user uses a bluetooth keyboard.

Reply:
My assumption here is that when menuKind is contextmenu it has the three dots that can be focused and clicked instead?

Reply:
Yes James is right. When the menuKind is 'contextual' then ellipses button will be present and keyboard accessible so we don't need to focus on the remote video tile. I don't think we want the keyboard user to tab on a remote video tile and then have to tab again to focus on the ellipses button to open the contextual menu:

https://user-images.githubusercontent.com/79475487/212398077-c67cd0ac-dc7b-4428-aef8-3271c8204cf6.mp4

But let me know if this is desired.